spec/support/tracer_helpers.rb

```diff
@@ -123,6 +125,11 @@ def low_order_trace_id(trace_id)
     Datadog::Tracing::Utils::TraceId.to_low_order(trace_id)
   end
 
+  ## Wraps call to Datadog::Tracing::Correlation.format_trace_id_128 for better test readability
+  def trace_id_128_log_injection(trace_id)
```

Comment:
Can we rename this to `format_for_correlation`? I think it succinctly captures what it's trying to do.